### ISSUE TITLE
Search "all" dropdown needs visual focus indicator

### ISSUE THREAD

**Issue by camillevilla** [sul-dlss/sul-bento-app#333]
## Chrome
![Chrome-focus-bento-banner](https://user-images.githubusercontent.com/5402927/89471753-cd8b8c00-d733-11ea-908b-19cf41daca41.gif)

## Firefox
![Firefox-focus-bento-banner](https://user-images.githubusercontent.com/5402927/89471760-cfede600-d733-11ea-8c4d-e65863999d6d.gif)

In both of these GIFs, I'm navigating with the keyboard and tab past the dropdown, then shift+tab to return to the dropdown and open it.

Though we can definitely open the "All" dropdown with keyboard nav, it doesn't have a visual focus indicator. This is also a concern for SearchWorks.

Some ideas for fixes (defer to @jvine):
1. change dotted underline to a solid underline
2. Add a background focus color to "all"

**Comment by jvine** [comment]
The best case would be to just turn the explicitly-turned-off (`#header-navbar .search-target:focus {outline: 0;}`) default outline back on, rather than create a new focus style. 

<img width="317" alt="Screen Shot 2020-08-05 at 5 13 37 PM" src="https://user-images.githubusercontent.com/6945691/89476238-07ae5b00-d73f-11ea-98fa-5ab579aab1bc.png">

But this won't immediately work in SearchWorks because of the styling of the button - the focus outline extends up to the top of the banner and looks weird.

<img width="429" alt="Screen Shot 2020-08-05 at 5 15 06 PM" src="https://user-images.githubusercontent.com/6945691/89476311-3c221700-d73f-11ea-8f64-4758c2b50962.png">

Which connects to an issue with the structure of the `<h1>` in the banner, which I will ticket separately.

**Comment by camillevilla** [comment]
@jvine I've deployed the outline fix from #334 to https://bento-stage.stanford.edu/. If that's sufficient let me know.

**Comment by camillevilla** [comment]
Closing in favor of sul-dlss/SearchWorks#2516
